mivs/site_sections/mivs_judging.py

```diff
@@ -0,0 +1,7 @@
+from mivs import *
+
+
```

Comment:
This is just a placeholder page for the judges/admins to look at the submissions.  We'll flesh it out in the coming weeks.